ethclient/ethclient.go

```diff
@@ -542,3 +542,51 @@ func toCallArg(msg ethereum.CallMsg) interface{} {
 	}
 	return arg
 }
+
+// rpcProgress is a copy of SyncProgress with hex-encoded fields.
+type rpcProgress struct {
```

Comment:
Shouldn't these have json tags?

Reply:
They didn't have before because all the fields work with the default mapping, and this struct is only used for decoding.